subsys/logging/log_backend_efi_console.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log_backend.h>
+#include <zephyr/logging/log_core.h>
+#include <zephyr/logging/log_output.h>
+#include <zephyr/logging/log_backend_std.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(log_efi);
```

Comment:
Not related to this PR, but I continue to be a little distressed at how much boilerplate is required for a log backend.

Reply:
Reduced the unnecessary include file. Thanks.